src/js/pages/Home/index.scss

```diff
@@ -21,8 +22,59 @@ $landing-text: #5C667D;
     padding-top: 20px;
   }
 
-  .home-headline {
-    margin-top: 1.5rem;
+  .home-headline-container {
+    width: 1146px;
+    text-align: center;
+    position: relative;
+    left: -231px;
+    margin-top: 60px;
+
+    @media screen and (max-width: 768px) {
+      width: auto;
+      left: 0;
+    }
+
+    &:before {
```

Comment:
Very clever!

Reply:
Thanks!
Not gonna lie, that was my girlfriend's idea hahaha. I asked her "what's the best way to have two background images" and she said "well, It's obvious, use before and after". I was like "yes, yes, very obvious :sweat_smile:"